drud/ddev
# Use alpine-based mariadb container

## The Problem/Issue/Bug:

This PR integrates a new alpine-based mariadb 10.1 container. It passes tests and works on Windows.

## How this PR Solves The Problem:

## Manual Testing Instructions:

Use regular ddev commands in many situations, on many OSs. 
Try `ddev rm` with and without -R and verify appropriate behavior.

## Automated Testing Overview:


## Related Issue Link(s):

OP: https://github.com/drud/ddev/issues/488
docker.mysql-local PR to incorporate this: https://github.com/drud/docker.mysql-local/pull/38
Earlier ddev PR explaining all the problems with debian-based mariadb: https://github.com/drud/ddev/pull/572
Earlier debian-based mariadb container attempt: https://github.com/drud/docker.mysql-local/pull/30


## Release/Deployment notes:
